### ISSUE TITLE
haversine error

### ISSUE THREAD

**Issue by jeff-hernandez** [FeatureLabs/featuretools#453]
### Bug/Feature Request Description
Haversine primitive raises an error in the following use case:
```python   
latlong1 = pd.Series({71: nan, 72: nan, 73: nan, 74: nan, 75: nan})
latlong2 = pd.Series({71: (40.798587799072266, -73.9694061279297), 72: (40.8021354675293, -73.96797943115234), 73: (40.77003860473633, -73.96251678466798), 74: (40.77336883544922, -73.96420288085938), 75: (40.73043823242188, -73.98625183105469)})
```
```
TypeError: 'float' object is not subscriptable
```

-----
*Issues created here on Github are for bugs or feature requests. For usage questions and questions about errors, please ask on Stack Overflow with the [featuretools](https://stackoverflow.com/questions/tagged/featuretools) tag. Check the [documentation](https://docs.featuretools.com/#get-help) for further guidance on where to ask your question.*

**Comment by vnarayan13** [comment]
Thinking of something like this to handle NaNs and scalars in the [haversine function](https://github.com/Featuretools/featuretools/blob/master/featuretools/primitives/standard/transform_primitive.py#L367):

`
def haversine(latlong1, latlong2):
    try:
        lat_1s, lon_1s = map(np.array, zip(*latlong1))
    except TypeError:
        lat_1s, lon_1s = [np.tile(np.nan, len(latlong1))]*2
    try:
        lat_2s, lon_2s = map(np.array, zip(*latlong2))
    except TypeError:
        lat_2s, lon_2s = [np.tile(np.nan, len(latlong2))]*2
`

**Comment by kmax12** [comment]
is there a way to vectorize the whole haversine primitive while also not erroring out on nans? any help on this is appreciated!

**Comment by ekerazha** [comment]
It looks like I got it working by representing LatLong missing values as `(np.nan, np.nan)`, however I still get this error when I use the `first` or `last` agg_primitives.

**Comment by ekerazha** [comment]
There's a pending PR about this issue: https://github.com/FeatureLabs/featuretools/pull/693

I don't know if it also fixes the problem with the `first` and `last` primitives.

**Comment by rwedge** [comment]
Thanks for bringing this up again @ekerazha!

Some extra info on how this happens with `first` and `last` primitives would be helpful.  Does this only happen with LatLong data?  Do you have an example of when this occurs?

**Comment by ekerazha** [comment]
@rwedge 
> Does this only happen with LatLong data?

This was basically my experience: I had missing LatLong values, represented as `np.nan`, and I got that error for haversine. So I tried to use `(np.nan, np.nan)` instead of `np.nan` and it fixed the haversine issue. Then I added all the available aggregation primitives instead of using the default ones and I got that error again. After removing the primitives one-by-one, I found that the `first` and `last` primitives caused the "new" issue.

**Comment by rwedge** [comment]
@ekerazha thanks for the info.  Do you have some example code I could run to try to reproduce this.  Also knowing what kind of environment you have would be helpful.  Running `ft.show_info()` in a python shell or `featuretools info` from a terminal would provide some useful info about your featuretools version, etc.  Thanks!

**Comment by thehomebrewnerd** [comment]
@ekerazha I believe the issue you were having should now be resolved with Featuretools v0.16.0 which was recently released. If you are still having issues with version 0.16.0, please let us know and we can reopen an issue.
